addons/crm/tests/test_crm_lead_assignment.py

```diff
@@ -533,3 +535,65 @@ def test_no_assign_if_exceed_max_assign(self):
         members_data = sales_team_4._assign_and_convert_leads()
         self.assertFalse(members_data,
             "If team member has lead count greater than max assign,then do not assign any more")
+
+    def test_assign_preferred_domain(self):
```

Comment:
Annoying guy remark: as tests are executed alphabeticall, better define them alphabetically in a given test class. That way when you rewrite something and have multiple failing tests, the reports in log match the order of tests in the file ^^ .